engine/src/dot/http_rpc.rs

```diff
@@ -224,4 +232,40 @@ mod tests {
 		let block_hash = dot_http_rpc.block_hash(1).await.unwrap();
 		println!("block_hash: {:?}", block_hash);
 	}
+
+	#[ignore = "Uses public mainnet polkadot endpoint"]
+	#[tokio::test]
+	async fn test_parsing_events_in_runtime_update_block() {
+		use std::str::FromStr;
+
+		// Block hash of the block that the runtime update occurred in
+		let block_hash_of_runtime_update =
+			H256::from_str("0xa0b52be60216f8e0f2eb5bd17fa3c66908cc1652f3080a90d3ab20b2d352b610")
+				.unwrap();
+
+		// The hash of the block before the runtime update block
+		let block_hash_of_parent =
```

Comment:
nit: we really only need to hardcode the runtime update hash, and we can get the parent hash by calling block(), makes it a little more flexible